services/movie.py

```diff
@@ -1,3 +1,4 @@
+from django.core.checks import translation
```

Comment:
The import `from django.core.checks import translation` seems incorrect. You likely intended to use `from django.db import transaction` for database atomic transactions.